screens/CompanyHomeScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import { View } from '../components/Themed';
+import TopBar from '../components/TopBar';
+import { ScrollView, StyleSheet } from 'react-native';
+import Notification from '../components/Notification';
+import React, { useEffect, useState } from 'react';
+import Flirt from '../types/Flirt';
+import backendFetch from '../lib/backendFetch';
+import Loading from '../components/Loading';
+
+export default function CompanyHomeScreen() {
+  const [flirts, setFlirts] = useState<Flirt[]>();
+  useEffect(() => {
+    backendFetch<Flirt[]>('GET', 'flirts')
+      .then(flirts => setFlirts(flirts as Flirt[]))
```

Comment:
Why the `as Flirt[]`? If you have it as a seperate function you can just tell typescript the return type is `Flirt[]`